js/map_utils.js

```diff
@@ -98,10 +97,13 @@ function zoomToFromState(d, i, j, selection) {
   }
   
   // zoom to the new view
-  updateView(newView);
+  updateView(newView, fireAnalytics = true);
 }
 
-function updateView(newView, fireAnalytics) {
+function getSessionId(){return new Date().getTime() + '.' + Math.random().toString(36).substring(5)}
```

Comment:
session ID is a timestamp?

Reply:
With some appended random characters.  It's just a way of ensuring a random number.

Reply:
To be more specific, it is a timestamp in milliseconds since 1970-01-01

Reply:
Cool, thanks :+1: